shared/chat/conversation/normal/container.js

```diff
@@ -20,6 +20,8 @@ const mapStateToProps = (state, {conversationIDKey}) => {
 }
 
 const mapDispatchToProps = dispatch => ({
+  _onPaste: (conversationIDKey: Types.ConversationIDKey, data: Buffer) =>
+    dispatch(Chat2Gen.createAttachmentPasted({conversationIDKey, data})),
   _onAttach: (conversationIDKey: Types.ConversationIDKey, paths: Array<string>) =>
     dispatch(
       RouteTree.navigateAppend([{props: {conversationIDKey, paths}, selected: 'attachmentGetTitles'}])
```

Comment:
Isn't this going to be broken? since paths here is an array of strings? Did you grep do see if anywhere else is navigating to `attachmentGetTitles`.

Since path is passed as a routeProp there really isn't any type checking to make sure that the `paths` you pass in to the route prop when you navigate is the correct type